### PR TITLE
Numerous updates

### PR DESCRIPTION
**Pull request by LetMePickThat**
- Changed max range for low altitude shots for most 48N6 and 9M82/3 missiles versions to better reflect the long-range capabilities of these missile (see: https://forums.eagle.ru/topic/269447-high-digit-sams-a-community-asset-pack-for-dcs-world/?do=findComment&comment=4677272),
- Added new 3D models for the SAMP/T and SAMP/T NG,
- Added a first version of the SAMP/T and SAMP/T NG,
- Updated S-300V and S-300VM 3D models,
- Updated default textures for the S-300V (now using a common three-tone late-1990s Russian camo),
- Added 3D models and .lua files for a first version of the S-400 (relevant entry.lua lines commented out to avoid releasing prematurely, the system still requires some work - adding the 9M96E/9M96E2 missiles for instance).
- Harmonized suffixes for trouble-prone 3D models (e.g 9M82.edm). Until now, these models used either the "_mod" or the "-hds" suffix. They now all use the latter.
- Edited entry.lua to reflect all those changes.

### CODE REVIEW

File: Mods/tech/HighDigitSAMs/entry.lua [hunk 1]
```diff
@@ -53,12 +53,14 @@ end
...
 dofile(current_mod_path.."/Database/Sensors/sensors.lua")
 
+
 -- Weapon files
 
 weapon_file("/Database/Weapon/5V55RUD.lua")
 weapon_file("/Database/Weapon/5V55R.lua")
 weapon_file("/Database/Weapon/48N6.lua")
 weapon_file("/Database/Weapon/48N6E2.lua")
+weapon_file("/Database/Weapon/48N6DM.lua")
```
Comment (Auranis) [inline Mods/tech/HighDigitSAMs/entry.lua:63]: This file is missing.

File: Mods/tech/HighDigitSAMs/entry.lua [hunk 2]
```diff
@@ -71,6 +73,8 @@ weapon_file("/Database/Weapon/9M342.lua")
 weapon_file("/Database/Weapon/9M36.lua")
 weapon_file("/Database/Weapon/ammunition.lua")
 weapon_file("/Database/Weapon/cannon.lua")
+weapon_file("/Database/Weapon/Aster_30_Blk_1NT.lua")
+weapon_file("/Database/Weapon/Aster_30_Blk_1.lua")
```
Comment (Auranis) [inline Mods/tech/HighDigitSAMs/entry.lua:77]: Aster 30 weapon files are missing.

File: Mods/tech/HighDigitSAMs/entry.lua [hunk 3]
```diff
@@ -145,7 +149,23 @@ vehicle_file("/Database/Vehicle/Polyana-D4M1.lua")
 vehicle_file("/Database/Vehicle/radar/34Ya6E Gazetchik E decoy.lua")
 
 
+-- S-400
```
Comment (Auranis) [inline Mods/tech/HighDigitSAMs/entry.lua:152]: New vehicle .luas are missing.

**Review by Auranis — CHANGES_REQUESTED** [review]
New .lua files referenced in the entry.lua are missing.

**Comment by LetMePickThat** [comment]
I wrote the requested changes.
All lines related to non-released systems are now removed.

**Pull request merged**